servers/plugins/org.fusesource.ide.server.karaf.core/src/org/fusesource/ide/server/karaf/core/server/subsystems/Karaf2xPublishController.java

```diff
@@ -211,6 +211,9 @@ public int transferBuiltModule(IModule[] module, IPath srcFile, IProgressMonitor
 			((Server)getServer()).setModulePublishState(module, IServer.PUBLISH_STATE_NONE);
 			((Server)getServer()).setServerPublishState(IServer.PUBLISH_STATE_NONE);			
 			status = IServer.PUBLISH_STATE_NONE;
+		} else {
+			Activator.getLogger().error("Deployment problem: Binary archive " + srcFile.toOSString() + " is not valid or lacking a valid MANIFEST.MF!");
```

Comment:
This sentence is the one provided in UI right?
Would be better to externalize it, if it is the case